src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -2634,6 +2634,40 @@ def test_aks_create_with_fips(self, resource_group, resource_group_location):
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    def test_aks_nodepool_add_with_artifact_streaming(self, resource_group, resource_group_location):
```

Comment:
Queued a [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=82516880&view=results) to validate the change.

Reply:
Thanks, this feature is for public preview and needs an AFEC feature flag registered (ArtifactStreamingPreview). As per the docs it will not pass the live test - is that accurate? So should I exclude the test cases by adding them to ext_matrix_default.json?